test_image.sh

```diff
@@ -223,6 +224,28 @@ function test_docker_image_scripts_2 {
 	test_page "http://${CONTAINER_HOSTNAME}:${CONTAINER_HTTP_PORT}/test_docker_image_scripts_2.jsp" "TEST2"
 }
 
+function test_docker_image_with_playwright {
+	export CONTAINER_EXPORTED_PORT=$(docker port "${CONTAINER_ID}" 8080 | head -n 1 | awk -F: '{print $2}')
+
+	if [ ! -d "playwright-dependencies" ]
+	then
+		echo "Directory playwright-dependencies does not exist"
+
+		exit 1
+	fi
+
+	cd playwright-dependencies || exit
+
+	local playwright_test_result=$(npx playwright test)
+
+	if [[ $playwright_test_result == *"1 passed"* ]]
```

Comment:
@lucasmiranda0, if tomorrow we add more tests, won't this if break? Can you make it a little more robust?